Casks/f/fontcreator.rb

```diff
@@ -0,0 +1,23 @@
+cask "fontcreator" do
+  version "15.0.0.3014"
+  sha256 :no_check
+
+  url "https://www.high-logic.com/FontCreator.zip"
```

Comment:
I think we can use the versioned url and then `sha256` can be calculated.
```suggestion
  url "https://www.high-logic.com/previousrelease/FontCreator#{version}.zip"
```